wpa-supplicant: update to 2.11

arkadijs opened the pull request:
Hi, updating [wpa_supplicant](https://w1.fi/wpa_supplicant/) to the latest version [released](https://w1.fi/cgit/hostap/plain/wpa_supplicant/ChangeLog) on Jul 20.

Also added a patch specific to proprietary `broadcom-wl` driver for PR #7212, sourced from Arch Linux https://gitlab.archlinux.org/archlinux/packaging/packages/wpa_supplicant/-/blob/main/0007-nl80211-add-extra-ies-only-if-allowed-by-driver.patch

Topic Description
-----------------

wpa-supplicant-2.11

Package(s) Affected
-------------------

app-network/wpa-supplicant

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

<!-- Please remove any architecture to which this topic does not apply. -->

Architectural progress for secondary ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

Architectural progress for experimental ports does not impede on merging of this topic.

- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`

OriginCode commented on app-network/wpa-supplicant/spec:
```diff
@@ -1,5 +1,5 @@
-VER=2.10
-REL=2
+VER=2.11
+REL=1
```
Our `REL` starts from 0, for an update like this you can simply remove the line as it defaults to 0.

OriginCode commented on app-network/wpa-supplicant/spec:
```diff
@@ -1,5 +1,5 @@
-VER=2.10
-REL=2
+VER=2.11
+REL=1
```
Our `REL` starts from 0, for an update like this you can simply remove the line as it defaults to 0.

arkadijs replied:
`REL` removed.

arkadijs committed:
wpa-supplicant: update to 2.11

Also add a patch specific to proprietary broadcom-wl driver, sourced from
https://gitlab.archlinux.org/archlinux/packaging/packages/wpa_supplicant/-/blob/main/0007-nl80211-add-extra-ies-only-if-allowed-by-driver.patch

arkadijs commented:
Force-pushed to make 1 commit - this repo cannot squash merge. Sorry for the trouble.

MingcongBai commented:
No worries Arkadi, thanks for your commit.
